### PR TITLE
fix(trading): fix symbol on order estimates

### PR DESCRIPTION
**Pull request by MadalinaRaicu**
# Related issues 🔗

Closes N/A

# Description ℹ️

Fix symbol on order estimates

# Demo 📺

Screenshot 2023-02-02 at 10 02 45

# Technical 👨‍🔧

Details of technical implementation that reviewers may need to be aware of, if applicable.

### CODE REVIEW

File: libs/deal-ticket/src/hooks/use-fee-deal-ticket-details.tsx [hunk 1]
```diff
@@ -69,12 +69,13 @@ export const useFeeDealTicketDetails = (
     return null;
   }, [derivedPrice, order.size, market.decimalPlaces]);
 
-  const quoteName = market.tradableInstrument.instrument.product.quoteName;
+  const symbol =
+    market.tradableInstrument.instrument.product.settlementAsset.symbol;
```
Comment (MadalinaRaicu) [inline libs/deal-ticket/src/hooks/use-fee-deal-ticket-details.tsx:73]: the issue was here. we were using quote name instead of asset symbol.

**Comment by campbellssource** [comment]
request (blocking): liquidation level should still be quote. not settlement asset.

**Review by asiaznik — APPROVED** [review]

File: libs/market-info/src/components/fees-breakdown/fees-breakdown.tsx [hunk 2]
```diff
@@ -115,7 +115,7 @@ export const FeesBreakdown = ({
         </dd>
       )}
       <dd className="text-right col-span-2">
-        {formatValue(totalFees)} {quoteName || ''}
+        {formatValue(totalFees)} {symbol || ''}
```
Comment (asiaznik) [inline libs/market-info/src/components/fees-breakdown/fees-breakdown.tsx:118]: symbol or quoteName :/

**Review by edd — APPROVED** [review]

File: libs/market-info/src/components/fees-breakdown/fees-breakdown.tsx (hunk 2 above)
Comment (MadalinaRaicu) [reply libs/market-info/src/components/fees-breakdown/fees-breakdown.tsx:118]: I am updating deal ticket price and liquidation to show quote name and I am leaving margin and fees in settlement asset symbol

**Pull request merged**